templates/supdfgen/vedtakAvslag.hbs

```diff
@@ -98,6 +132,24 @@
             med kort botid i Norge {{> supdfgen/partials/visAvslagsparagrafer avslagsparagrafer}}.
         </span>
 
+        {{#each avslagsgrunner}}
+            {{#eq this "OPPHOLDSTILLATELSE"}}
+                {{#if erAldersbrev}}
+                    <p>Personer med registreringsbevis kan søke om varig oppholdsrett etter fem års botid i Norge.
+                        Retten til supplerende stønad kan tidligst oppnås når du har fått vedtak om varig oppholdsrett.
```

Comment:
Er dette helt sant? Ref. https://www.udi.no/ord-og-begreper/oppholdstillatelse-som-danner-grunnlag-for-permanent-oppholdstillatelse/

og:
>Ein person vert rekna som busett dersom han eller ho er registrert i folkeregisteret og har norsk statsborgarskap, eller med heimel i utlendingslova har fått permanent opphaldsløyve eller mellombels opphaldsløyve som gir grunnlag for permanent opphaldsløyve

Reply:
Jeg tolker det slik, ref https://www.udi.no/ord-og-begreper/varig-oppholdsrett/

Reply:
Ja, ser i rundskrivet også at formuleringene rundt oppholdsrett ikke er helt presise. Jeg følger det opp!

Reply:
Kan du bare merge denne og lage et eget kort på det?